tests/unittests/IROptTest.cpp

```diff
@@ -125,6 +125,33 @@ TEST(Optimizer, shareBuffers) {
   EXPECT_EQ(M.getInstrs().size(), 2);
 }
 
+TEST(Optimizer, deleteDeadViews) {
+  Module mod;
+  Function *F = mod.createFunction("DeleteDeadViews");
+  IRFunction M(F);
+  IRBuilder bb(&M);
+
+  auto *input = bb.createWeightVar(glow::ElemKind::FloatTy, {1}, "input",
+                                   WeightVar::MutabilityKind::Constant);
+  auto *output = bb.createWeightVar(glow::ElemKind::FloatTy, {1}, "ouput",
+                                    WeightVar::MutabilityKind::Mutable);
+
+  auto tensor_view1 = bb.createTensorViewInst(
```

Comment:
Is this `createTensorViewInst` clang-formatted? The indentation looks a bit strange.